code/datums/quirks/_quirk_static_data.dm

```diff
@@ -0,0 +1,74 @@
+GLOBAL_LIST_INIT_TYPED(all_quirk_static_data, /datum/quirk_static_data, generate_quirk_static_data())
+
+/// Constructs [GLOB.all_quirk_static_data] by iterating through a typecache of pregen data, ignoring abstract types, and instantiating the rest.
+/proc/generate_quirk_static_data()
```

Comment:
You should rename this to constant data because static data has a very specific meaning